front/src/pages/Category.tsx

```diff
@@ -351,11 +390,12 @@ const CloseFilterButton = styled(Button)`
   }
 `
 
+type SortOrder = 'asc' | 'desc'
```

Comment:
This is old code!

Reply:
Here’s the new version: [Category.tsx](https://github.com/IT-Academy-BCN/ita-wiki/blob/main/front/src/pages/Category.tsx)